.circleci/config.yml

```diff
@@ -15,10 +15,13 @@ defaults: &defaults
   - checkout
   - run:
       name: Install dependencies
-      command: pip install -r requirements.txt
+      command: pip install tox
```

Comment:
We're trying to move away from using CircleCI and towards github actions. Would you mind holding off any changes to CircleCI config?